python/lsst/cell_coadds/_cell_coadd_builder.py

```diff
@@ -98,13 +98,12 @@ def run(
 
         Parameters
         ----------
-        inputs: `Mapping[ObservationIdentifiers, tuple[DeferredDatasetHandle,
-                                                       lsst.geom.Box2I]]`
-            A mapping from `lsst.cell_coadds.ObservationIdentifiers`` to a
+        inputs: `Mapping[ObservationIdentifiers, tuple[DeferredDatasetHandle, lsst.geom.Box2I]]`  # noqa: W505
```

Comment:
```suggestion
        inputs: `~collections.abc.Mapping [ `ObservationIdentifiers`, \
            `tuple` [ `~lsst.daf.butler.DeferredDatasetHandle`, \
            `~lsst.geom.Box2I`] ]
```
- backticks around every symbol
- qualify some symbols
- us backslashes and extra indentation for line breaks (but I'm not sure I put them in the right place, as I can't see number-of-columns in the PR easily)

Reply:
Recall that package-docs now has a nitpick mode that can tell you lots of fun issues with your docstrings (you will need to tell it to download the pipelines.lsst.io index for referencing other packages). You can also add "D" (pydocstyle) to your ruff configuration.

Reply:
Thanks, I'm going to do this in #24 and merge that one before this PR.

Reply:
This is still going to exceed 79 characters and having `#noqa: W505` to please the linters appears on the built documentation.

Reply:
Yeah, I didn't check that I put the escaped newlines in the right place in my suggestion.  But you can add more until it fits in 79 chars, since no single symbol here is too long.

Reply:
I think the [docstring in the code](https://github.com/lsst/cell_coadds/blob/main/python/lsst/cell_coadds/_cell_coadd_builder.py#L101-L102) itself is wrapped fine (AFAIK), but this is how they appear on the single-package documentation build. Even the symbols in this package aren't linked correctly. Unhappy, but not too much because I see this going away when we move out the responsibility of building it to `drp_tasks` (the same time when I expect `pipe_tasks` dependency to go away)

<img width="522" alt="image" src="https://github.com/lsst/cell_coadds/assets/5103390/8a3f176f-7289-4f6b-ac6f-4a7026588cc6">

Reply:
To clarify on that point, I don't just mean to say that this method will migrate to a different package, but I don't see us building one cell at a time (and thus this abstractmethod). We're likely going to be reading in a warp and do online coaddition of whatever cells they contribute.

Reply:
To get the butler and geom names to link properly in the local package build you need to add the pipelines.lsst.io object.inv to your doc/conf.py.

Reply:
In the example docstring there is a missing backtick after "Mapping" and that is confusing the parser I think. Did nitpick mode not report this problem?

Reply:
I fixed that missing backtick and no errors/warnings when I run `package-docs build -n`. (I'm not asking it to download references from other packages though)

Reply:
I'm a bit confused because I get 430 errors when I run `package-docs build -n` (also, why is doxygen being run in the build?)

Anyhow, your problem is that you are missing the space between the parameter name and the colon.

Reply:
D'oh! 🤦🏼